plugins/GSdx/GSDrawScanlineCodeGenerator.x86.cpp

```diff
@@ -2889,7 +2889,7 @@ void GSDrawScanlineCodeGenerator::ReadTexel_SSE(int pixels, int mip_offset)
 				movdqa(ptr[&m_local.temp.test], xmm7);
 			}
 
-			for(int j = 0; j < 4; j++)
+			for(uint8 j = 0; j < 4; j++)
```

Comment:
You might be bringing a bit of extra overhead by indexing via 8-bit variable, see here:

http://www.viva64.com/en/a/0050/

Do note I have **not** compared assembly outputs for this specific case and it's probably a negligible difference, but depending on how performance critical this code is, it might be a good practice to let go and opt for a natural word size variable instead.